versioned_hdf5/api.py

```diff
@@ -150,3 +156,32 @@ def stage_version(self, version_name: str, prev_version=None,
         except:
             delete_version(self.f, version_name, old_current)
             raise
+
+    def close(self):
+        """
+        Make sure the VersionedHDF5File object is no longer reachable.
+        """
+        if not self._closed:
+            self.__del__()
```

Comment:
Should it call self.f.close()?

Reply:
I was not sure if this should be done silently.